packages/shared_preferences/shared_preferences/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: shared_preferences
 description: Flutter plugin for reading and writing simple key-value pairs.
   Wraps NSUserDefaults on iOS and SharedPreferences on Android.
 homepage: https://github.com/flutter/plugins/tree/master/packages/shared_preferences/shared_preferences
-version: 0.5.6
+version: 0.5.7
```

Comment:
Bump the patch version as there is no new feature in this change (this should be 0.5.6+1)

Reply:
done